test/ImportCommand.js

```diff
@@ -84,10 +84,10 @@ describe("ImportCommand", () => {
       expect(await lastCommitInDir(externalDir)).toBe("Branch merged");
 
       await lernaImport(externalDir, "--flatten");
-      expect(await lastCommitInDir(testDir)).toBe("Branch merged");
+      expect(await lastCommitInDir(externalDir)).toBe("Branch merged");
```

Comment:
The `"Branch merged"` commit is no longer imported into the target repo? That doesn't seem right.

Reply:
Dug through code, and I didn't see in the tests where you were actually adding those files to `testDir`, just saw them being created in `externalDir`. Is there a function that's supposed to copy those over in this test context?

Reply:
When I ran tests on `master` (with *no changes*), I was getting Promise errors, and errors for the tests dealing with `testDir`.